src/components/SideBar.astro

```diff
@@ -88,3 +88,31 @@
     </div>
   </div>
 </div>
+<script>
+  const homeLink = document.getElementById('home');
+  const projectsLink = document.getElementById('projects');
+  const servicesLink = document.getElementById('services');
+  const storeLink = document.getElementById('store');
+  const blogLink = document.getElementById('blog');
+  const cvLink = document.getElementById('cv');
+
+  var isHome = true;
+
+  const url = window.location.href;
+
+  const pages = [ projectsLink, servicesLink, storeLink, blogLink, cvLink ];
+
+  pages.forEach( (elem) => {
+    if (url.includes(elem.id)) {
+      isHome = false;
+      elem.classList.add('bg-neutral');
+      elem.classList.add('text-white');
+      homeLink.classList.remove('bg-neutral');
+      homeLink.classList.remove('text-white');
+    } 
+    if (isHome == true) {
```

Comment:
Here we can move this if out of the for and remove the == and just leave the var that is a boolean